client/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
+        "node": true,
+        "browser": true,
+        "es6": true
+    },
+    "plugins": [
+        "react"
+    ],
+    "extends": [
+        "airbnb",
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "parser": "babel-eslint",
+    "rules": {
```

Comment:
We had to disable certain rules. I'll go through them.

1. We disabled "react/jsx-filename-extension". This linting rule makes sure you need to use '.jsx' file names instead of '.js'.

2. We disabled some indenting rules (jsx-indent and jsx-indent-props). I'll got back through and fix this up later

3. We disabled react/prop-types. I don't think we should emphasis prop types right now because we gotta push this out.

Reply:
Please take note of these.  We want to re-enable react/prop-types so that it is more readable (we know what is being passed in) and what each type of prop should be

Reply:
Yeah for sure that makes sense. I think it we can re-enable the rule once we've set up some basic propTypes for every component.